### DISCUSSION TITLE
{Mojo} | Playground 体验 | Notebook 多图 | 附代码链接

### DISCUSSION THREAD

**Discussion by ToughStyle** [mojo-cn/.github#3] [Show and tell]
体验账号今早被批准了, 分享一下感受, 最后一张图遇到了泛型指针bug, 有待官方改进
本篇是基础部分, 后面从编译时元编程开始的高级特性期待大神探索
有过批体验账号的朋友可以猛戳这个链接验证结果: https://playground.modular.com/hub/user-redirect/lab/tree/Custom.ipynb
欢迎多讨论交流
![1](https://github.com/mojo-cn/.github/assets/35249976/a32ae55c-cbbe-499a-ae0e-0772937741bb)
![2](https://github.com/mojo-cn/.github/assets/35249976/d8c92948-3222-4d61-9d9d-4b58a250f210)
![3](https://github.com/mojo-cn/.github/assets/35249976/7f1ab393-b81a-4a76-ab06-0eb8d1b9f2f5)
![4](https://github.com/mojo-cn/.github/assets/35249976/0c6e7f6e-7f15-4245-a2d8-985e60d5a129)

**Comment by hhstore** [comment]
赞, 顺便可以把  `jupyter notebook` 源文件发出来. 

后续, 可以基于你的测试代码, 验证 & 测试. 😄️
